onnxruntime/contrib_ops/cpu/quantization/qlinear_softmax.cc

```diff
@@ -37,11 +38,12 @@ void QlinearBuildLookupTableUint32(gsl::span<uint32_t> table,
     // if is_signed index = [1 2 3 ......126 127 -128 -127 ..... -3 -2 -1]
     // else [0 1 2 3 4 ..... 256]
     uint8_t index = static_cast<uint8_t>(is_signed ? i - 128 : i);
-    table[index] = static_cast<uint32_t>(lrint(scaled_exp_xi));
+    table[index] = static_cast<QLinearSoftmax::EXP_OUT_DTYPE>(std::nearbyint(scaled_exp_xi));
```

Comment:
Essentially, it is same as the original. What's the difference?

Reply:
The original implementation rounds half away from zero as ONNX usually uses rounding half to even. To keep it, we may change it to: `const uint32_t vq = std::nearbyint(float(vt) * float(c_y_scale) / float(vsum)) + c_y_zp;`

https://github.com/microsoft/onnxruntime/blob/64e991a9fc044004859913a7e5d0940e93f703ee/onnxruntime/contrib_ops/cpu/quantization/qlinear_softmax.cc#L180